app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -704,7 +705,7 @@ class AdministratorControlsActivityTest {
       WorkManagerConfigurationModule::class, HintsAndSolutionConfigModule::class,
       FirebaseLogUploaderModule::class, FakeOppiaClockModule::class, PracticeTabModule::class,
       DeveloperOptionsStarterModule::class, DeveloperOptionsModule::class,
-      ExplorationStorageModule::class
+      ExplorationStorageModule::class, NetworkModule::class
```

Comment:
Could you help me understand why are we adding this?

Reply:
@jcqli Do you think this is right? 
I am not seeing it right by importing data module stuff in the app module. 

@rt4914 Could you suggest a better architecture around here? 

@ARJUPTA Could you please add the error log here when we run the test without it?

Reply:
Hm thanks for catching this I'm also curious. Does the `WorkManagerConfigurationModule` need the `NetworkModule`?

Reply:
@jcqli @anandwana001 have replied to your doubt here -> https://github.com/oppia/oppia-android/pull/3606#discussion_r687384854